modules/social_features/social_featured_content/templates/group--featured.html.twig

```diff
@@ -86,7 +91,8 @@
   {% if group_members is not empty %}
     <div class="badge teaser__badge">
       <span class="badge__container">
-        <svg class="badge__icon" aria-hidden="true" title="{% trans %}Number of group members{% endtrans %}">
+        <svg class="badge__icon">
```

Comment:
This SVG should be hidden because there's an sr-only line with the same information right after it.

Reply:
Fixed